UPGRADE.md

```diff
@@ -7,6 +7,9 @@
 * In order to be compatibile with Doctrine ORM 2.6+ and be more consistent 
   `OrderRepositoryInterface::count()` signature was changed to `OrderRepositoryInterface::countPlacedOrders()`.
 
+* In statistics to be correct data with orders informations 
```

Comment:
How about `In statistics to display information about fulfilled orders` instead?